docs/reference/search/profile.asciidoc

```diff
@@ -164,12 +153,26 @@ But occasionally multiple searches will be executed, such as including a global
 a secondary "match_all" query for the global context).
 
 Inside each `search` object there will be two arrays of profiled information:
-a `query` array and a `collector` array.  In the future, more sections may be added, such as `suggest`, `highlight`,
-`aggregations`, etc
+a `query` array and a `collector` array.  Alongside the `search` object is a `aggregations` object that contains the profile information for the aggregations. In the future, more sections may be added, such as `suggest`, `highlight`, etc
 
 There will also be a `rewrite` metric showing the total time spent rewriting the query (in nanoseconds).
 
-=== `query` Section
+=== Profiling Queries
```

Comment:
actually that's how it is today, and there is quite a lot of info, so i'd leave it as it is